templates/base.html.twig

```diff
@@ -428,7 +428,7 @@
                             </li>
                             <li>
                                 <a href="https://legislatives.en-marche.fr" target="_blank" rel="noopener noreferrer">
```

Comment:
i know it's not related, but for legislatives routes we should now use the url generator:
`<a href="{{ path('legislatives_homepage') }}" target="_blank" rel="noopener noreferrer">`